pkgs/by-name/bu/bulk_extractor/package.nix

```
@@ -0,0 +1,51 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, exiv2
+, flex
+, libewf
+, libxml2
+, openssl
+, tre
+, zlib
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "bulk_extractor";
+  version = "2.0.3";
+
+  src = fetchFromGitHub {
+    owner = "simsong";
+    repo = "bulk_extractor";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-T1EOa+HehmCytD9ZQx/F6TNEXt7mMNWu7tqYPZII3N0=";
+    fetchSubmodules = true;
+  };
+
+  # https://github.com/simsong/bulk_extractor/pull/434
+  postPatch = ''
+      substituteInPlace src/Makefile.am \
+            --replace "bin_PROGRAMS   = bulk_extractor test_be" "bin_PROGRAMS   = bulk_extractor"
+  '';
+
+  enableParallelBuilding = true;
+  nativeBuildInputs = [autoreconfHook];
+  buildInputs = [ exiv2 flex libewf libxml2 openssl tre zlib ];
```

```suggestion
  nativeBuildInputs = [ autoreconfHook flex ];
  buildInputs = [ exiv2 ibewf libxml2 openssl tre zlib ];
```
flex is ususally run on the host to generate files